showcase/blueprints/hds-component-test/index.js

```diff
@@ -4,7 +4,7 @@
  */
 
 /* eslint-disable ember/no-string-prototype-extensions */
-/* eslint-disable node/no-extraneous-require */
+/* eslint-disable n/no-extraneous-require */
```

Comment:
😮

Reply:
`eslint-plugin-node` has been unmaintained for a while and this is the best way the community could come up with, so it's being replaced with `eslint-plugin-n`